### PR TITLE
Formatting fixes

### PR DESCRIPTION
**Pull request by banilo**

### CODE REVIEW

File: hcp/io/read.py [hunk 1]
```diff
@@ -586,6 +590,7 @@ def _read_evoked(fname, sensor_mode, info):
     info['sfreq'] = sfreq
 
     out = list()
+    # this look really hacky!
```
Comment (dengemann) [inline hcp/io/read.py]: but it's not. it's just compact and ok since naming patterns are set forever.

File: hcp/io/read.py [hunk 2]
```diff
@@ -215,6 +215,10 @@ def read_info_hcp(subject, data_type, run_index=0, hcp_path=op.curdir):
     -------
     info : instance of mne.io.meas_info.Info
         The MNE channel info object.
+
+    Note
+    ----
+    HCP MEG does not deliver only 3 of the 5 task packages from MRI HCP.
```
Comment (dengemann) [inline hcp/io/read.py]:
```
.. note::
   HCP MEG does only deliver 3 of the 5 task packages from MRI HCP.
```

File: hcp/io/read.py [hunk 3]
```diff
@@ -75,7 +75,7 @@ def _read_trans_hcp(fname, convert_to_meter):
 
 
 def _read_landmarks_hcp(fname):
-    """ XXX parse landmarks currently not used """
+    """XXX parse landmarks currently not used"""
```
Comment (dengemann) [inline hcp/io/read.py]: ok for this type of white space deletion.

File: hcp/io/read.py (hunk 2 above)
Comment (jona-sassenhagen) [inline hcp/io/read.py]: HCP MEG only deliver**s** 3 of the 5 task packages from MRI HCP.

**Comment by dengemann** [comment]
Thanks @banilo ! Can someone check if there are more of these STY fixes to do? Open issue or PR.

**Pull request merged**